Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -79,6 +79,10 @@ instance coeSubmodule : CoeOut (LieSubmodule R L M) (Submodule R M) :=
   ⟨toSubmodule⟩
 #align lie_submodule.coe_submodule LieSubmodule.coeSubmodule
 
+instance instCanLiftSubmoduleLieSubmodule : CanLift (Submodule R M) (LieSubmodule R L M) (·)
```

Comment:
How about:
```suggestion
instance instCanLiftSubmoduleLieSubmodule : CanLift (Submodule R M) (LieSubmodule R L M) (↑)
```

Reply:
Those are not syntactically equal, for some reason. And I think the `.toSubmodule` version provided by `(·)` has better simp properties.

We should probably fix the coe at some point.